### PR TITLE
DCKUBE-662: Release 1.0.0

### PR DESCRIPTION
**Pull request by nghazali**

### CODE REVIEW

File: src/main/charts/jira/Changelog.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## 1.0.0
+
+![AppVersion: 8.13.9-jdk11](https://img.shields.io/static/v1?label=AppVersion&message=8.13.9-jdk11&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+* DCKUBE-621: Improve graceful shotdown (#282)
```
Comment (nanux) [inline src/main/charts/jira/Changelog.md:9]:
```suggestion
* DCKUBE-621: Improve graceful shutdown (#282)
```

File: src/main/charts/jira/Chart.yaml [hunk 2]
```diff
@@ -21,34 +21,29 @@ deprecated: false
...
   artifacthub.io/containsSecurityUpdates: "true"
   artifacthub.io/changes: |
-    - kind: added
-      description: Enable NFS permission fixer by default
-      links:
-      - name: Github PR
-        url: https://github.com/atlassian/data-center-helm-charts/pull/241
     - kind: changed
-      description: Enable configuration for SET_PERMISSIONS docker image variable
+      description: Improve graceful shotdown
```
Comment (nanux) [inline src/main/charts/jira/Chart.yaml:25]:
```suggestion
      description: Improve graceful shutdown
```

File: src/main/charts/bitbucket/Changelog.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+
+## 1.0.0
+
+![AppVersion: 7.15.1-jdk11](https://img.shields.io/static/v1?label=AppVersion&message=7.15.1-jdk11&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+* DCKUBE-621: Improve graceful shotdown (#282)
```
Comment (nanux) [inline src/main/charts/bitbucket/Changelog.md:10]:
```suggestion
* DCKUBE-621: Improve graceful shutdown (#282)
```

File: src/main/charts/bitbucket/Changelog.md [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+* DCKUBE-658: Use the AWS LB Controller with ALB/NLBs instead of nginx for testing (#294)
```
Comment (nanux) [inline src/main/charts/bitbucket/Changelog.md:11]: We can drop this one from the changelog, it is just for internal testing.

File: src/main/charts/bitbucket/Chart.yaml [hunk 5]
```diff
@@ -19,39 +19,29 @@ sources:
...
-    - kind: added
-      description: Configurable grace periods
+      description: Improve graceful shotdown
       links:
       - name: Github PR
-        url: https://github.com/atlassian/data-center-helm-charts/pull/249
+        url: https://github.com/atlassian/data-center-helm-charts/pull/282
     - kind: changed
-      description: Update the Bitbucket image name, as the '-server' suffix is now deprecated
+      description: Use the AWS LB Controller with ALB/NLBs instead of nginx for testing
```
Comment (nanux) [inline src/main/charts/bitbucket/Chart.yaml:28]: For testing only, we can drop it.

File: src/main/charts/bitbucket/Changelog.md [hunk 6]
```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+
+## 1.0.0
```
Comment (nanux) [inline src/main/charts/bitbucket/Changelog.md:4]: I think it is worth calling out that this is the first supported version:

> This is the first officially supported version of the helm chart.

Or something like that.

**Review by nanux — CHANGES_REQUESTED** [review]

File: src/main/charts/bitbucket/Changelog.md (hunk 6 above)
Comment (nghazali) [reply src/main/charts/bitbucket/Changelog.md:4]: done

File: src/main/charts/confluence/Changelog.md [hunk 7]
```diff
@@ -1,4 +1,16 @@
 # Change Log
+This is the first officially supported version of the helm chart.
```
Comment (nanux) [inline src/main/charts/confluence/Changelog.md:2]: This should be under the 1.0.0 version and should be in all product changelogs.

**Review by nanux — CHANGES_REQUESTED** [review]

**Review by nanux — APPROVED** [review]

File: src/main/charts/bitbucket/Changelog.md [hunk 8]
```diff
@@ -1,5 +1,17 @@
...
+This is the first officially supported version of the helm chart.
```
Comment (errcode1202) [inline src/main/charts/bitbucket/Changelog.md:5]: should `helm chart` be `Helm Charts`? If so same change throughout.

File: src/main/charts/bitbucket/Chart.yaml [hunk 9]
```diff
@@ -19,39 +19,24 @@ sources:
...
-      - name: Github PR
-        url: https://github.com/atlassian/data-center-helm-charts/pull/261
-    - kind: added
-      description: Configurable grace periods
-      links:
-      - name: Github PR
-        url: https://github.com/atlassian/data-center-helm-charts/pull/249
     - kind: changed
-      description: Update the Bitbucket image name, as the '-server' suffix is now deprecated
+      description: Improve the graceful shutdown
```
Comment (errcode1202) [inline src/main/charts/bitbucket/Chart.yaml:23]: Should probably be: `Improvements to graceful shutdown`. And throughout.

File: src/main/charts/confluence/Changelog.md [hunk 10]
```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+
+## 1.0.0
+This is the first officially supported version of the helm chart.
```
Comment (errcode1202) [inline src/main/charts/confluence/Changelog.md:5]: should probably be `Helm Charts`?

**Review by badgersow — APPROVED** [review]

**Review by errcode1202 — CHANGES_REQUESTED** [review]

File: src/main/charts/bitbucket/Changelog.md (hunk 8 above)
Comment (yzha645) [inline src/main/charts/bitbucket/Changelog.md:5]: `Helm chart` instead of `helm chart`?

Comment (nanux) [reply src/main/charts/bitbucket/Changelog.md:5]: It is probably `Helm chart` as it is a changelog for one chart? Not sure about the `C` vs `c`, @tan-ro ?

File: src/main/charts/bitbucket/Changelog.md [hunk 11]
```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+
+## 1.0.0
+This is the first officially supported version of the Helm Charts.
```
Comment (yzha645) [inline src/main/charts/bitbucket/Changelog.md:5]: `charts` should probably be lowercase to be consistent with our other documentation.

File: src/main/charts/bitbucket/Changelog.md (hunk 8 above)
Comment (nghazali) [reply src/main/charts/bitbucket/Changelog.md]: I see we have `Helm chart` in all our documentation so I followed the existing format here.

**Review by errcode1202 — APPROVED** [review]

**Review by yzha645 — APPROVED** [review]

File: src/main/charts/bitbucket/Changelog.md (hunk 11 above)
Comment (errcode1202) [reply src/main/charts/bitbucket/Changelog.md]: Hmm @tan-ro whats the go on this? I thought `Helm Charts`?

**Pull request merged**